dask/base.py

```diff
@@ -23,6 +23,41 @@ class Base(object):
 
     def visualize(self, filename='mydask', format=None, optimize_graph=False,
                   **kwargs):
+        """
+        Render the computation of this object's task graph using graphviz.
+
+        Requires `graphviz` to be installed.
```

Comment:
One last nitpick: sphinx doesn't like single backticks. All places where you do `` `graphviz` `` can you do ``` ``graphviz`` ```?